README.md

```diff
@@ -16,3 +16,18 @@ evaluation of air quality and atmospheric composition.
 ## Documentation
 
 Built and hosted by Read the Docs: <https://melodies-monet.readthedocs.io>
+
+# Disclaimer
+
+The United States Department of Commerce (DOC) GitHub project code is provided
```

Comment:
```suggestion
This United States Department of Commerce (DOC) GitHub project code is provided
```
I know it was pulled from GDFL, but this makes more sense to me.

Reply:
Looks like this is very official language, so let's leave it as is: https://github.com/CommerceGov/Policies-and-Guidance/blob/master/GithubGuidanceforDepartmentofCommerce.md. But we can update the quotes since we are trying to be consistent with quotes across the repository.